src/pages/search/searchResults.tsx

```diff
@@ -2,8 +2,10 @@ import { Box, VStack, Heading } from "@chakra-ui/react";
 import React from "react";
 import { Project } from "../../../server/src/db/entities/Project";
 import { ProjectCard } from "../../components/projects/ProjectCard";
+import { UserCard } from "../../components/UserCard"
+import Profile from "../../models/profile";
 
-export default function SearchResults({ results }): JSX.Element {
+export default function SearchResults({ results, fetching }): JSX.Element {
```

Comment:
export default line should be at the end of the file with a new line break after it